packages/nimble-components/src/select/specs/filterable-select-hld.md

```diff
@@ -55,12 +84,16 @@ export class LabelProviderCore
 
     @attr({ attribute: 'select-filter-no-results' })
     public selectFilterNoResults: string | undefined;
+
+    @attr({ attribute: 'select-loading-options' })
+    public selectLoadingOptions: string | undefined;
```

Comment:
discussed with meyers and seems like this and the above other labels, "selectFilterSearch", "selectFilterNoResults", "selectLoadingOptions" are being overly coupled to the select and could be more general concepts. The name proposed here is okay if we do not want a breaking change right now. If we can handle a breaking change we should consider "filterSearch", "filterNoResults", and "loading" for more general applicability (i.e. combobox, rich-text user mention, table filter, etc.)

Reply:
I'm at least going to just have the new attribute be called `loading`. We can leave the name changes for the other attributes for another time/PR.